src/client/src/rt-platforms/openFin/adapter/openFin.ts

```diff
@@ -66,27 +59,8 @@ export default class OpenFin extends BasePlatformAdapter {
   }
 
   window = {
-    close: () => fin.desktop.Window.getCurrent().close(),
```

Comment:
this has been removed in favor of two things - creating a new window with OpenFin API and controlling current window with wrapping it into `PlatfromWindow`